PSReadLine.build.ps1

```diff
@@ -32,6 +32,10 @@ Write-Verbose "Building for '$target'" -Verbose
 
 $CLI_VERSION = "2.1.300"
 
+function ConvertTo-CRLF([string] $text) {
```

Comment:
@SteveL-MSFT Sure seems like such a basic, cross-platform support utility should be built-into PowerShell.  :-)  Along with ConvertTo-LF.   Or maybe just `Convert-LineEnding` with `-CRLF` and `-LF` exclusive switches.

Reply:
@rkeithhill We do have https://github.com/PowerShell/PowerShell/issues/2872